src/azure-cli/HISTORY.rst

```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+**IoT**
+
+* Set default SKU to S1 while hub create.
+
```

Comment:
Instead of writing HISTORY.rst manually, we have automatically generated history through PR title and description. 
Please delete the content here and modify the PR title and description. For details, please refer to: [doc link](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests)

Reply:
Updated the title